libraries/AP_Common/Location.cpp

```diff
@@ -86,6 +86,14 @@ bool Location::change_alt_frame(AltFrame desired_frame)
     return true;
 }
 
+void Location::copy_alt_from(const Location &other)
```

Comment:
Should we be copying terrain relative altitudes without locations? That is somehow loosing some of the information. I note there is only one user in any-case.

Reply:
Good call, added an internal error

Reply:
... which we then removed at DevCall :-)